docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1159,12 +1153,12 @@ The following error message codes may appear on the app-server log. Use the foll
   The NODE_PATH is %s
 
   **Reason:**
-
-  TODO
+  The NODE_PATH is *_process.env.NODEPATH_*.
+  NODE_PATH is set to environment.
```

Comment:
Remove both lines

*The server recognizes the location of Node as _NODE_PATH_ from environment variable.